### DISCUSSION TITLE
Order of collections within the left sidebar

### DISCUSSION THREAD

**Discussion by lt-bha** [GitbookIO/community#383] [0. General bugs and questions]
Is there a way to move collections to a specific position within the sidebar on the left? Currently the collections are listed at the bottom. Even within the collections they cannot be moved to the desired order.

**Comment by aikoii** [comment]
👋 @lt-bha, currently, collections can't be moved above spaces in the sidebar; it's by design. However, you still should be able to reorder collection within collections. Could you  [hard-refresh](https://docs.gitbook.com/help/hard-refresh) your browser to get the latest version of the app and try again?
